### PR TITLE
Updating 2018-03-01-preview to include a few extra fields and parameters

### PR DESCRIPTION
**Pull request by sendhil**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json [hunk 1]
```diff
@@ -475,7 +475,8 @@
           "GetEntities": {
             "$ref": "./examples/GetEntities.json"
           }
-        }
+        },
+        "x-ms-long-running-operation": true
       }
```
Comment (hovsepm) [inline specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json]: does this POST operation also have 202 response?

File: specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json [hunk 2]
```diff
@@ -1363,7 +1388,7 @@
       "schema": {
         "$ref": "#/definitions/CreateManagementGroupRequest"
       },
-      "x-ms-parameter-location": "method"
+      "x-ms-parameter-location": "client"
```
Comment (hovsepm) [inline specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json]: >client [](start = 34, length = 6)

why is this parameter changed from method location to client? By doing this change you require developers to set global parameter on the client instance and not on the method parameters.

File: specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json [hunk 3]
```diff
@@ -1373,7 +1398,7 @@
       "schema": {
         "$ref": "#/definitions/PatchManagementGroupRequest"
       },
-      "x-ms-parameter-location": "method"
```
Comment (hovsepm) [inline specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json]: Same question here - why is the location changed from method signature to client object?

File: specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json [hunk 4]
```diff
@@ -1430,7 +1457,7 @@
         "delete"
       ],
       "description": "The id of the operation result.",
-      "x-ms-parameter-location": "method"
+      "x-ms-parameter-location": "client"
```
Comment (hovsepm) [inline specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json]: >client [](start = 34, length = 6)

please revert all these changes for method location.

**Comment by hovsepm** [comment]
@sendhil could you provide some information on why do you add so many line breaks? are you trying to fix any particular documentation rendering issue? Swagger should be UI agnostic service endpoint description. Please remove all `\n` special characters from the PR.

**Review by hovsepm — CHANGES_REQUESTED** [review]
Please address review comments.

**Comment by rajshah11** [comment]
@hovsepm I will close this PR and make a new one from my forked repo. Will make the changes in the comments too. Thanks!

**Comment by hovsepm** [comment]
Closed per owner team request. /cc @rajshah11

**Pull request closed without merging**